calvinpete/WillGen
# ch(requirements.txt): upgrade requirements.txt file

#### What does this PR do?
This PR updates the packages required for the application to run smoothly on another computer.

#### Description of Task to be completed?
Install the listed packages in the requirements.txt file before running the app.

#### How should this be manually tested?
- First things first, fetch this branch by running the command `git fetch  ch-upgrade-requirements.txt-180606797` in the terminal.
- Check into this branch by running `git checkout  ch-upgrade-requirements.txt-180606797`
- Activate your virtual environment by running `source venv/bin/activate`
- Install the requirements by running `pip install -r requirements.txt`

#### What are the relevant pivotal tracker stories?
[#180606797](https://www.pivotaltracker.com/story/show/180606797)
